server/src/db/seeders/ReviewSeeder.js

```diff
@@ -4,30 +4,42 @@ class ReviewSeeder {
   static async seed() {
     const reviewData = [
       {
-        rating: "10",
+        rating: 10,
         description: "I have never seen a better dog. Seriously.",
-        upvotes: 5,
-        downvotes: 1,
         userId: 1,
         dogId: 2,
       },
       {
-        rating: "9",
+        rating: 9,
         description: "Wow.  What a dog!",
-        upvotes: 3,
-        downvotes: 2,
         userId: 3,
         dogId: 1,
       },
       {
-        rating: "14",
+        rating: 14,
         description:
           "I have been looking at pictures of dogs for over 30 years. This is the best-looking one yet.",
-        upvotes: 5,
-        downvotes: 1,
         userId: 2,
         dogId: 3,
       },
+      {
+        rating: 13,
+        description: "This is a fairly average dog. Not impressed.",
+        userId: 1,
+        dogId: 2,
```

Comment:
Instead of "arbitrary" `id` values here, ideally we should be querying the object that we want to make associations with

Reply:
Refactored DogSeeder and Review Seeder so as not to rely on hard-coded ids